src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -86,10 +101,16 @@ public Map<CreationEnum, List<? extends AzureAccount>> addAzureAccounts(
             try {
                 User user = azureUserService.createUser(azureAccount);
                 azureAccount.setAzureAccountId(user.id);
-                createdAzureAccounts.add(azureAccount);
 
                 log.info(writeLog(issuerEmail, UserActions.CREATE_ACCOUNT, azureAccount.getEmail()));
-                handleAccountCreationEmail(azureAccount);
+
+                if (!handleAccountCreationEmail(azureAccount, isExisting)) {
```

Comment:
Wondering if this should be reversed - create account then sent email. Just so users arn't sent emails for their account if it has not been set up?

Reply:
ah yeh thats totally my bad, we even said we'd do that 😂 updated to add at original point, not under condition of email